biodatasets/med_qa/med_qa.py

```diff
@@ -0,0 +1,213 @@
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+
+        urls = _URLS[_DATASETNAME]
+        data_dir = dl_manager.download_and_extract(urls)
+        lang_dict = {"en": "US", "zh": "Mainland", "tw": "Taiwan"}
+        base_dir = os.path.join(data_dir, "data_clean", "questions")
+        if self.config.subset_id in ["med_qa_en", "med_qa_zh", "med_qa_tw"]:
+            lang_path = lang_dict.get(self.config.subset_id.rsplit("_", 1)[1])
+            paths = {
+                "train": os.path.join(base_dir, lang_path, "train.jsonl"),
+                "test": os.path.join(base_dir, lang_path, "test.jsonl"),
+                "valid": os.path.join(base_dir, lang_path, "dev.jsonl"),
+            }
+        elif self.config.subset_id == "med_qa_tw_en":
+            paths = {
+                "train": os.path.join(base_dir, "Taiwan", "tw_translated_jsonl", "en", "train-2en.jsonl"),
+                "test": os.path.join(base_dir, "Taiwan", "tw_translated_jsonl", "en", "test-2en.jsonl"),
+                "valid": os.path.join(base_dir, "Taiwan", "tw_translated_jsonl", "en", "dev-2en.jsonl"),
+            }
+        elif self.config.subset_id == "med_qa_tw_zh":
+            paths = {
+                "train": os.path.join(base_dir, "Taiwan", "tw_translated_jsonl", "zh", "train-2zh.jsonl"),
+                "test": os.path.join(base_dir, "Taiwan", "tw_translated_jsonl", "zh", "test-2zh.jsonl"),
+                "valid": os.path.join(base_dir, "Taiwan", "tw_translated_jsonl", "zh", "dev-2zh.jsonl"),
+            }
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepath": paths["train"],
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.TEST,
+                gen_kwargs={
+                    "filepath": paths["test"],
+                },
+            ),
+            datasets.SplitGenerator(
+                name=datasets.Split.VALIDATION,
+                gen_kwargs={
+                    "filepath": paths["valid"],
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples."""
+        print(filepath)
+        data = pd.read_json(filepath, lines=True)
+
+        if self.config.schema == "source":
+            for key, example in data.iterrows():
+                example = example.to_dict()
+                example["options"] = [{"key": key, "value": value} for key, value in example["options"].items()]
+                yield key, example
+
+        elif self.config.schema == "bigbio_qa":
+            for key, example in data.iterrows():
+                example = example.to_dict()
+                example_ = {}
+                example_["id"] = str(key)
+                example_["question_id"] = ""
```

Comment:
Could you please use the same value as "id" for the `question_id` and `document_id`? I quickly skimmed through merged QA [dataloaders](https://github.com/bigscience-workshop/biomedical/blob/master/biodatasets/pubmed_qa/pubmed_qa.py#L247), and this seems to be the way that folks dealt with this. Thank you for working on this!

Reply:
Sure, I've changed it!